### PR TITLE
Merging view : terminals adapter implementation

### PR DESCRIPTION
**Pull request by tadam50**
Signed-off-by: Thomas ADAM <tadam@silicom.fr>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
New adaptated classes are implemented.
Each method adaptated not implemented yet throw an exception

**What is the current behavior?** *(You can also link to an open issue here)*
Merging view module implementation need to implement terminal adapters

**What is the new behavior (if this is a feature change)?**
Merging view module implementation step : Terminal adapters implementation needed

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalAdapter.java [hunk 1]
```diff
@@ -16,81 +16,96 @@
  */
 public class TerminalAdapter extends AbstractAdapter<Terminal> implements Terminal {
 
+    private TerminalBusBreakerViewAdapter busBreakerView;
+    private TerminalBusViewAdapter busView;
+    private TerminalNodeBreakerViewAdapter nodeBreakerView;
```
Comment (mathbagu) [inline iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalAdapter.java]: Same question than in the previous PR: should we keep the views as neested class to fit the IIDM API?

File: iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalAdapter.java [hunk 2]
```diff
@@ -16,81 +16,96 @@
...
     @Override
     public boolean isConnected() {
-        throw MergingView.NOT_IMPLEMENTED_EXCEPTION;
+        return getDelegate().isConnected();
     }
 
     @Override
     public void traverse(final TopologyTraverser traverser) {
-        throw MergingView.NOT_IMPLEMENTED_EXCEPTION;
+        getDelegate().traverse(traverser);
```
Comment (mathbagu) [inline iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalAdapter.java]: Not sure about traverse... Revert this change: we already talk about this, and we have to think more how to wrap correctly a traverser.

File: iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalNodeBreakerViewAdapter.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package com.powsybl.iidm.mergingview;
```
Comment (mathbagu) [inline iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalNodeBreakerViewAdapter.java]: Add license and author information

File: iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalAdapter.java (hunk 1 above)
Comment (tadam50) [reply iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalAdapter.java]: I will make theses fields as nested classes

File: iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalAdapter.java (hunk 2 above)
Comment (tadam50) [reply iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalAdapter.java]: I will revert this change

File: iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalNodeBreakerViewAdapter.java (hunk 3 above)
Comment (tadam50) [reply iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalNodeBreakerViewAdapter.java]: I will add header to this file

Comment (tadam50) [reply iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalNodeBreakerViewAdapter.java]: This class will be deleted
